### PR TITLE
chore(radio): Cleanup startup code.

### LINKED ISSUE #4018: TX 16 S EdgeTX 2.9 How to remove the vibrator when switching on the transmitter?
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Even choosing in the radio settings "None" for the vibrate, it still works when turning on the transmitter...

### Describe the solution you'd like

I would like to have to choose in the radio parameters to completely disable the vibrator (even for turning on (or off) the transmitter)

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by philmoz**
Fixes #4018 

Summary of changes:
- Remove SPLASH build option
- Move splash handling code to gui files/folders
- Rename opentxXXX functions to edgeTxXXX
- Fix throttle warning startup timing (haptic played before splash screen finished on color LCD)
- Use radio setting to control startup/shutdown haptic

### CODE REVIEW

**Comment by MRC3742** [comment]
I have tested this branch on a T16 and works as described.
Startup/down haptic is properly suppressed when ONLY "Quiet" mode is selected.

Just a thought for possible addition to this PR.
(1) Shut off Startup/down haptic when EITHER the "Alarm" or "Quiet" mode is selected?
(2) Or could an option be added, under the haptic menu, to turn it off/on at startup?

Thank You - Rich

haptic

**Comment by LS-67** [comment]
In which version of ETX will this modification be implemented?

**Comment by pfeerick** [comment]
@LS-67 2.10

Would the suggestion by Rich which implemented in this PR as no haptic on startup or shutdown if set to Quiet be sufficient for you? As that could _possibly_ be added for 2.9.

**Comment by LS-67** [comment]
This will suit me very well ! Thank's a lot !

**Review by raphaelcoeffic — APPROVED** [review]
Ok for me, @3djc please review the STDLCD part.

File: radio/src/gui/128x64/radio_setup.cpp [hunk 1]
```diff
@@ -188,7 +181,7 @@ void menuRadioSetup(event_t event)
     CASE_IMU(LABEL(IMU))
     CASE_IMU(0)
     CASE_IMU(0)
-    LABEL(ALARMS), 0, CASE_CAPACITY(0)
```
Comment (3djc) [inline radio/src/gui/128x64/radio_setup.cpp:191]: Shouldn't the target section be deleted too then ?

#if defined(TX_CAPACITY_MEASUREMENT)
      case ITEM_RADIO_SETUP_CAPACITY_WARNING:
        lcdDrawTextAlignedLeft(y, STR_CAPAWARNING);
        drawValueWithUnit(LCD_W-2, y, g_eeGeneral.mAhWarn*50, UNIT_MAH, attr|RIGHT) ;
        if(attr) CHECK_INCDEC_GENVAR(event, g_eeGeneral.mAhWarn, 0, 100);
        break;
#endif

Comment (philmoz) [reply radio/src/gui/128x64/radio_setup.cpp:191]: Thanks. I've removed this code plus associated code from a few other files. Also removed unused strings only used by this code.

**Review by pfeerick — APPROVED** [review]
LGTM on TX16S, X9D+, Pocket

**Pull request merged**